partner_affiliate/models/res_partner.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2012 Camptocamp SA - Yannick Vaucher
+# Copyright 2018 brain-tec AG - Raul Martin
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    """Add relation affiliate_ids."""
+    _inherit = "res.partner"
+
+    child_ids = fields.One2many(domain=[('is_company', '=', False)])
```

Comment:
Even though not mandatory, its good to specify string attribute

Reply:
I am even in the favour of writing full details of the field like you did for affilate_ids.